src/Social/Mapper/AbstractMapper.php

```diff
@@ -62,6 +62,7 @@ public function __construct($mapFields = null)
         }
         $this->_mapFields = array_merge($this->_defaultMapFields, $this->_mapFields);
     }
+
```

Comment:
What's the reason to add these empty lines?

Reply:
phpcbf did it for us to match conventions